crates/bevy_core_pipeline/src/prepass/mod.rs

```diff
@@ -63,6 +69,22 @@ pub struct MotionVectorPrepass;
 #[derive(Component, Default, Reflect)]
 pub struct DeferredPrepass;
 
+#[derive(Component, ShaderType, Clone)]
+pub struct PreviousViewData {
+    pub inverse_view: Mat4,
```

Comment:
Just a note that since this: https://github.com/bevyengine/bevy/pull/13489
is in the 0.14 milestone it might get in soon and then these names would likely need to be updated to match